### PR TITLE
Table and Resolution Select Improvements

### PR DESCRIPTION
**Pull request by incognitojam**
I've edited the table so that the column titles are more descriptive, and hovering over the acronyms results in a small OS-independant pop-up.

I've also added some more common screen resolutions - I noticed a few were missing that I've used in the past.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,10 +74,10 @@
 			<thead>
 				<tr>
 					<th>Name</th>
-					<th>Diagonal</th>
+					<th>Diagonal (inches)</th>
 					<th>Resolution</th>
-					<th>DPI</th>
-					<th>dppx</th>
+					<th><span title="Dots Per Inch">DPI</span></th>
```
Comment (LeaVerou) [inline index.html]: Why not add the tooltip on the `th` directly? We don’t need a separate element here.

File: index.html [hunk 2]
```diff
@@ -72,10 +74,10 @@
 			<thead>
 				<tr>
 					<th>Name</th>
-					<th>Diagonal</th>
+					<th>Diagonal (inches)</th>
```
Comment (LeaVerou) [inline index.html]: This will stretch the size of the column unnecessarily. Better as a tooltip.

**Comment by LeaVerou** [comment]
Thanks! See inline comments.

File: index.html (hunk 2 above)
Comment (incognitojam) [inline index.html]: Thanks for the comment - I've made the changes.

File: index.html (hunk 1 above)
Comment (incognitojam) [inline index.html]: I've made the changes accordingly. (Sorry, I'm a bit under-experience with HTML).

**Comment by incognitojam** [comment]
I've now made all of the changes you suggested.

File: index.html [hunk 3]
```diff
@@ -26,8 +26,10 @@
 	<p id="resolutions">Common: 
 		<a href="#" tabindex="-1">1920×1080</a>
 		<a href="#" tabindex="-1">1680×1050</a>
+		<a href="#" tabindex="-1">1680×900</a>
 		<a href="#" tabindex="-1">1440×900</a>
 		<a href="#" tabindex="-1">1366×768</a>
+		<a href="#" tabindex="-1">1360×768</a>
```
Comment (LeaVerou) [inline index.html:32]: I would be opposed to adding new predefined resolutions that are so similar to existing examples. But even if I wasn't, it's always a good practice to send separate PRs for separate changes.

**Comment by LeaVerou** [comment]
Thanks! See inline comment.

**Comment by LeaVerou** [comment]
Since the resolutions haven't been removed to this PR, I'm afraid I can't merge it. :(
Closing for now, for housekeeping.
Please reopen or send a new PR for the HTML changes, those were very welcome.
